### ISSUE TITLE
AWS not fetching S3 buckets

### ISSUE THREAD

**Issue by ahmednrana** [cloudquery/cloudquery#22]
I created a config using 
`cloudquery gen config aws`
followed by
`cloudquery fetch`
I also added this in config.yml
```   regions: # Optional. if commented out assumes all regions
     - us-east-1
     - us-west-2
     - eu-west-1
     - eu-west-2```
The output of command is 
```
.\cloudquery_Windows_x86_64.exe fetch
2020-12-28T15:03:15.392+0500    INFO    Creating tables if needed       {"provider": "aws"}
2020-12-28T15:03:15.438+0500    INFO    No regions specified in config.yml. Assuming all 20 regions     {"provider": "aws"}
2020-12-28T15:03:28.254+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "region": "eu-west-1", "resource": "fsx.backups", "count": 0}
2020-12-28T15:03:28.303+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "region": "eu-west-1", "resource": "auto_scaling.launch_configurations", "count": 0}
2020-12-28T15:03:28.336+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "region": "eu-west-1", "resource": "elasticbeanstalk.environments", "count": 0}
2020-12-28T15:03:28.357+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "region": "eu-west-1", "resource": "efs.filesystems", "count": 0}
2020-12-28T15:03:28.391+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "region": "eu-west-1", "resource": "rds.certificates", "count": 1}
2020-12-28T15:03:28.437+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "region": "eu-west-1", "resource": "rds.clusters", "count": 0}
2020-12-28T15:03:28.459+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "region": "eu-west-1", "resource": "rds.subnet_groups", "count": 1}
2020-12-28T15:03:28.465+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "resource": "iam.groups", "count": 2}
2020-12-28T15:03:28.469+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "region": "eu-west-1", "resource": "cloudtrail.trails", "count": 0}
2020-12-28T15:03:28.470+0500    INFO    Fetched resources       {"provider": "aws", "account_id": "333870179331", "region": "eu-west-1", "resource": "emr.clusters", "count": 0}
2020/12/28 15:03:28 NoSuchEntity: The Password Policy with domain name 333870179331 cannot be found.
        status code: 404, request id: ef255e51-7a8c-4e85-aa68-d926500dfbe1
```

I have S3 buckets in different regions but none are showing by using the command 
`SELECT * from aws_s3_buckets `
The table is empty

**Comment by korsune** [comment]
@ahmednrana Thanks for reporting! I believe this should be fixed in [v0.6.4](https://github.com/cloudquery/cloudquery/releases/tag/v0.6.4)

**Comment by ahmednrana** [comment]
Working fine now.
